microprofile-reactive-messaging-kafka/enable-reactive-messaging.cli

```diff
@@ -0,0 +1,5 @@
+/extension=org.wildfly.extension.microprofile.reactive-messaging-smallrye:add
+/extension=org.wildfly.extension.microprofile.reactive-streams-operators-smallrye:add
+/subsystem=microprofile-reactive-streams-operators-smallrye:add
+/subsystem=microprofile-reactive-messaging-smallrye:add
```

Comment:
These should ideally be run in a batch/run-batch

Reply:
@rhusar fixed